src/main/java/frc/robot/commands/AlignToColumn.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.pathplanner.lib.PathPoint;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.controls.Operator;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Go to 1-9 column in grid area
+ */
+public class AlignToColumn extends SequentialCommandGroup {
+
+  private final Drivetrain m_drive;
+  
+  public AlignToColumn(Drivetrain drive) {
+    m_drive = drive;
+    addRequirements(drive);
+    addCommands(
+      new PathPlannerCommand(new ArrayList<PathPoint>(List.of(
+        PathPoint.fromCurrentHolonomicState(m_drive.getPose(), m_drive.getChassisSpeeds()),
+        new PathPoint(Operator.selectedNode.scorePose.getTranslation(), Operator.selectedNode.scorePose.getRotation())
```

Comment:
this is reliant on Operator please make it so it is not relient on operator as we are removeing for now.

Reply:
This command doesn't work without Operator. Also, it doesn't cause an error because I didn't comment out the selectedNode in Operator.

Reply:
you can make it non-dependent on operator by adding a node paramater